ui/src/views/Profile/index.jsx

```diff
@@ -37,6 +37,9 @@ export default class Profile extends Component {
       <Dashboard title="Profile">
         {!currentScopes && loading && <Spinner loading />}
         <ErrorPanel fixed error={error} />
+        <Typography variant="subtitle1">
+          Sign in to view your Profile
```

Comment:
"Profile" can be lower case.

Reply:
This is still not done. `profile` instead of `Profile`.

Reply:
Sorry for that, it's done now!